commit 158b5ed504a03b4fedd8b58cabcbdd1c464e378f
Author: Tanvir Roshid <tanvirroshid786123@gmail.com>
Date:   Tue Jan 14 18:47:30 2025 +0000

    files/linux/fdsdk-config.sh: Reorganise and document
    
    This commit reorganises the existing PinePhone Pro kernel configurations
    to group related configs logically, and add comments to document each one.
    
    There are no functional changes introduced.
    
    (cherry picked from commit c9b02326aad00f8e55823e16aa9f3362f5ab1172)

diff --git b/files/linux/fdsdk-config.sh a/files/linux/fdsdk-config.sh
index ffabc9769..3dc8dd3d8 100644
--- b/files/linux/fdsdk-config.sh
+++ a/files/linux/fdsdk-config.sh
@@ -10,40 +10,33 @@ arch=$1
 # PinePhone Pro kernel configs
 case "$arch" in
     aarch64)
-        # Camera Drivers
-        module VIDEO_ROCKCHIP_ISP1       # Rockchip Image Signal Processing (ISP) support
-        module VIDEO_ROCKCHIP_RGA        # Rockchip Raster 2D Graphic Acceleration Unit
-        module VIDEO_ROCKCHIP_VDEC       # Rockchip Video Decoder driver
-        module VIDEO_HANTRO              # Hantro VPU (Video Processing Unit) driver
-        module VIDEO_OV8858              # OmniVision OV8858 camera sensor support
-        module VIDEO_IMX258              # Sony IMX258 camera sensor support
-        module VIDEO_DW9714              # DW9714 camera lens voice coil driver
-
-        # Graphics and Display
-        enable ROCKCHIP_RGB              # Rockchip RGB support
-        module DRM_PANEL_HIMAX_HX8394    # Himax HX8394 display panel support
-
-        # Thermal and Power Management
-        module ROCKCHIP_THERMAL          # Thermal sensor support for Rockchip SoCs
-        enable ROCKCHIP_MBOX             # Inter-processor communication support for Rockchip SoCs
-
-        # Audio/Video Enhancements
-        module V4L2_FLASH_LED_CLASS      # V4L2 flash API support for LED flash
-
-        # Cryptography
-        module CRYPTO_DEV_ROCKCHIP       # Rockchip's Cryptographic Engine driver
-
-        # Connectivity and Miscellaneous Hardware
-        module PHY_ROCKCHIP_DPHY_RX0     # Rockchip MIPI Synopsys DPHY RX0 driver
-        module ROCKCHIP_SARADC           # SAR A/D Converter bindings for Rockchip SoCs
-
-        # Input Devices
-        module INPUT_GPIO_VIBRA          # GPIO-based vibrator device support
-        module KEYBOARD_PINEPHONE        # PinePhone keyboard case support
-
-        # Backlight and LED
-        enable BACKLIGHT_CLASS_DEVICE    # Low-level backlight control
-        module LEDS_SGM3140              # SGM3140 LED driver (500mA Buck/Boost Charge Pump)
+        module VIDEO_ROCKCHIP_ISP1
+        module VIDEO_ROCKCHIP_RGA
+        module VIDEO_ROCKCHIP_VDEC
+        module VIDEO_HANTRO
+        module VIDEO_OV8858
+        module VIDEO_IMX258
+        module VIDEO_DW9714
+
+        module ROCKCHIP_SARADC
+        enable ROCKCHIP_MBOX
+        module ROCKCHIP_THERMAL
+        enable ROCKCHIP_RGB
+
+        module CRYPTO_DEV_ROCKCHIP
+
+        module PHY_ROCKCHIP_DPHY_RX0
+
+        enable BACKLIGHT_CLASS_DEVICE
+
+        module V4L2_FLASH_LED_CLASS
+
+        module INPUT_GPIO_VIBRA
+        module KEYBOARD_PINEPHONE
+
+        module DRM_PANEL_HIMAX_HX8394
+
+        module LEDS_SGM3140
     ;;
 esac
 
